src/assets/styles/_component.scss

```diff
@@ -0,0 +1,61 @@
+:host {
```

Comment:
We could consider this being something we extend so that components could easily overwrite the background and there would be no two `:host` selectors.

Reply:
That makes sense.

Reply:
Would it work as a `_host` partial? `_component-host`?

Reply:
Just a thought.